regtests/search.test.js

```diff
@@ -0,0 +1,34 @@
+import elastic from '../backend/elastic';
+import Keys from '../common/Keys';
```

Comment:
Definitely need to flesh out these tests, but before we have a concrete idea of what that might look like, I think this is a good start.

Reply:
Looks great!